rules_java_gapic/java_gapic.bzl

```diff
@@ -372,10 +372,11 @@ def java_gapic_library(
         **kwargs
     )
 
-def java_gapic_test(name, runtime_deps, test_classes, **kwargs):
+def java_gapic_test(name, runtime_deps, test_classes, size = "small", **kwargs):
```

Comment:
Is this change used anywhere?

Reply:
It prevents warning messages from appearing in the bazel builds. Otherwise, no.